force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -11837,6 +11837,14 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>pmtPaidPaymentDateRequired</shortDescription>
         <value>A paid Payment must have a Payment Date.</value>
     </labels>
+    <labels>
```

Comment:
@sam-knox Can you review the text on the newly updated UI? The UI changed from a single sentence to a table view because the currency conversion makes it impossible to translate correctly.
![Screen Shot 2022-05-25 at 11 23 18 AM](https://user-images.githubusercontent.com/57271120/170338827-157d55a9-ba61-4d3c-90ec-f74d19e6f2cb.png)
.